tests/MoneyTest.php

```diff
@@ -584,6 +584,7 @@ public function roundToUnitExamples(): array
             [1250, 2, 1200, Money::ROUND_DOWN],
             [1250, 2, 1300, Money::ROUND_HALF_POSITIVE_INFINITY],
             [1250, 2, 1200, Money::ROUND_HALF_NEGATIVE_INFINITY],
+            [1, 2, 0, Money::ROUND_HALF_UP],
```

Comment:
try any value from 10-99 cents

Reply:
10-49 indeed fails, 99 does work.

Reply:
erg. i should try my examples before posting :D